### DISCUSSION TITLE
can no longer see untracked files in stashes

### DISCUSSION THREAD

**Discussion by waveforms** [gitkraken/vscode-gitlens#1889] [Q&A]
Using VScode version 1.65.0 and the latest git lens that shipped this week, I can no longer see untracked files in my stashes. I used this feature heavily before, often creating new files and stashing them if I had to switch away. Now when I go back to my stash list I cannot tell which stash has the new untracked files.  I have set showIncludeUntracked in my local git config file to be true but still cannot see them. Is this a bug or is there a setting in git lens I should adjust? Unfortunately I lost a couple of days work on this but I did manage to learn a lot more about the git.
Thanks for this useful extension. Anybody else having this issue?

VScode:
Version: 1.65.0
Commit: b5205cc8eb4fbaa726835538cd82372cc0222d43
Date: 2022-03-02T11:11:20.886Z
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.3.0

**Comment by eamodio** [comment]
This has been fixed in the latest version.

**Comment by shamseer-ahammed** [comment]
+
